docs/src/main/sphinx/release/release-401.md

```diff
@@ -5,23 +5,35 @@
 * Add support for using path-style access for all requests to S3 when using
   fault-tolerant execution with exchange spooling. This can be enabled with the
   `exchange.s3.path-style-access` configuration property. ({issue}`14655`)
+* Add support for polymorphic table functions in file-based access control. ({issue}`13713`)
 * Add distribution data for output buffer utilization to
   `EXPLAIN ANALYZE VERBOSE`. ({issue}`14596`)
 * Add distribution data for operator blocked time to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14640`)
 * Improve performance and reliability of `INSERT`, `MERGE`, and table optimization. ({issue}`14553`)
+* Fix query failure caused by a
+  `com.google.common.base.VerifyException: cannot unset noMoreSplits` error. ({issue`14668`})
+* Fix underestimation of CPU usage and scheduled time statistics for the build
```

Comment:
Maybe 

Correct estimation of

Reply:
I'd rather leave it as-is. I don't think that's really any clearer, and this maintains the pattern of using 'fix' as the first word to describe fixes.

Reply:
fine by me

Reply:
"the build side" is a term of art that most people won't understand. I would simplify to "for joins"